internal/service/advancedcluster/resource_advanced_cluster_migration_test.go

```diff
@@ -109,12 +109,12 @@ func TestMigAdvancedCluster_shardedMigrationFromOldToNewSchema(t *testing.T) {
 		Steps: []resource.TestStep{
 			{
 				ExternalProviders: acc.ExternalProviders(versionBeforeISSRelease),
-				Config:            configShardedTransitionOldToNewSchema(orgID, projectName, clusterName, false),
+				Config:            configShardedTransitionOldToNewSchema(orgID, projectName, clusterName, false, false),
 				Check:             checkShardedTransitionOldToNewSchema(false),
 			},
 			{
 				ProtoV6ProviderFactories: acc.TestAccProviderV6Factories,
-				Config:                   configShardedTransitionOldToNewSchema(orgID, projectName, clusterName, true),
+				Config:                   configShardedTransitionOldToNewSchema(orgID, projectName, clusterName, true, false),
 				Check:                    checkShardedTransitionOldToNewSchema(true),
```

Comment:
Can we assert autoscaling mode in the `shardedTransitionFromOldToNewSchema` tests (both migration and acceptance)? The expected outcome would be that it goes from CLUSTER to SHARD, even if autoscaling is not enabled.